apps/github-monday-code/src/storage/base-storage.js

```diff
@@ -0,0 +1,45 @@
+import { Storage } from '@mondaycom/apps-sdk';
+
+class BaseStorage {
+  constructor(token) {
+    this.storage = new Storage(token);
+  }
+  /**
+   * @typedef {Object} Options
+   * @property {string} [token] - The token value.
+   * @property {string} [previousVersion] - The previous version.
+   */
```

Comment:
I assume this is a mistake and should be deleted, correct? - These are the options for the set

Reply:
i see all operations have these options, in https://github.com/mondaycom/apps-sdk

<img width="724" alt="Screen Shot 2023-07-19 at 11 46 31" src="https://github.com/mondaycom/welcome-apps/assets/105207523/8ae6c393-9b65-4b86-8364-d9586516da9c">